src/templates/cc-link/cc-link.js

```diff
@@ -75,4 +75,8 @@ export const linkStyles = css`
     background-color: var(--cc-color-text-primary-weak, hsl(209deg 98% 73%));
     color: transparent;
   }
+
+  .cc-link cc-icon {
+    margin-right: 0.25em;
+  }
```

Comment:
**question:** is it this still needed and are we sure it doesn't impact placed we don't know about?
Since you are using `display: flex` within documentation links, I would rely on `gap` and I wouldn't make `cc-link` itself add the margin (but it's not a bad idea overall, it's just hard to track which components / cases we impact so I would be cautious and wait for a `cc-link` component to introduce this kind of change).

Reply:
Nice catch ! I did not think it  could have impact on other places than those components. I have removed it and add a gap directly on the `.cc-link` concerned, thank you !